node/src/Commands.ts

```diff
@@ -848,6 +848,18 @@ export function createZInterstore(
     return createCommand(RequestType.ZInterStore, args);
 }
 
+/**
+ * @internal
+ */
+export function createZUnionstore(
```

Comment:
Please rename to follow the common style
```suggestion
export function createZUnionStore(
```